dom0/sd-dom0-qvm-rpc.sls

```diff
@@ -146,10 +70,23 @@ dom0-rpc-qubes.r5-format-ask-allow:
         securedrop.Log          *           sd-log sd-log deny notify=no
         securedrop.Log          *           @tag:sd-workstation sd-log allow
 
+        securedrop.Proxy        *           sd-app sd-proxy allow
+
+        qubes.Gpg               *           @tag:sd-client sd-gpg allow
+
+        qubes.USBAttach         *           sys-usb sd-devices allow  user=root
+        qubes.USBAttach         *           @anyvm @anyvm ask
+
+        qubes.USB               *           sd-devices sys-usb allow
+
+        # TODO: should this be handled with the new Global Config UI instead?
```

Comment:
I think this is appropriate for the policy file, so that the config is set by files we control (and if `uninstall` is run it will be removed cleanly)